documentation/Quickstart-on-how-to-write-tests.md

```diff
@@ -10,6 +10,13 @@ This page is to help you write and run tests quickly for Javascript Codegen SDK
   - [Code structure](#code-structure)
   - [Run tests in record mode](#run-tests-in-record-mode)
   - [Run tests in playback mode](#run-tests-in-playback-mode)
+  - [How to push test recordings to assets repo](#how-to-push-test-recordings-to-assets-repo)
```

Comment:
Please refer below to add any missing part in prerequisites section

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md#prerequisites